types/hapi/index.d.ts

```diff
@@ -1,2489 +1,2599 @@
-		encoding: string;
-		/**  - an object used to calculate an HMAC for cookie integrity validation.This does not provide privacy, only a mean to verify that the cookie value was generated by the server.Redundant when 'iron' encoding is used.Options are:*/
-		sign: {
-			/** - algorithm options.Defaults to require('iron').defaults.integrity.*/
-			integrity: any;
-			/** - password used for HMAC key generation.*/
-			password: string;
-		};
-		/**  - password used for 'iron' encoding.*/
-		password: string;
-		/** - options for 'iron' encoding.Defaults to require('iron').defaults.*/
-		iron: any;
-		/** - if false, errors are ignored and treated as missing cookies.*/
-		ignoreErrors: boolean;
-		/**  - if true, automatically instruct the client to remove invalid cookies.Defaults to false.*/
-		clearInvalid: boolean;
-		/**  - if false, allows any cookie value including values in violation of RFC 6265. Defaults to true.*/
-		strictHeader: boolean;
-		/**  - overrides the default proxy localStatePassThrough setting.*/
-		passThrough: any;
-	};
-}
-
-export interface IFileHandlerConfig {
-	/**  a path string or function as described above.*/
-	path: string;
-	/** an optional filename to specify if sending a 'Content-Disposition' header, defaults to the basename of path*/
-	filename?: string;
-	/**- specifies whether to include the 'Content-Disposition' header with the response. Available values:
-	 false - header is not included. This is the default value.
-	 'attachment'
-	 'inline'*/
-	mode?: boolean | string;
-	/** if true, looks for the same filename with the '.gz' suffix for a pre-compressed version of the file to serve if the request supports content encoding. Defaults to false.*/
-	lookupCompressed: boolean;
-}
-
-/**http://hapijs.com/api#route-handler
- Built-in handlers
-
- The framework comes with a few built-in handler types available by setting the route handler config to an object containing one of these keys.*/
-export interface IRouteHandlerConfig {
-	/** generates a static file endpoint for serving a single file. file can be set to:
-	 a relative or absolute file path string (relative paths are resolved based on the route files configuration).
-	 a function with the signature function(request) which returns the relative or absolute file path.
-	 an object with the following options */
-	file?: string | IRequestHandler<void> | IFileHandlerConfig;
-	/** directory - generates a directory endpoint for serving static content from a directory. Routes using the directory handler must include a path parameter at the end of the path string (e.g. /path/to/somewhere/{param} where the parameter name does not matter). The path parameter can use any of the parameter options (e.g. {param} for one level files only, {param?} for one level files or the directory root, {param*} for any level, or {param*3} for a specific level). If additional path parameters are present, they are ignored for the purpose of selecting the file system resource. The director
```

Comment:
I don't think this field is mandatory, just updated typings and this is causing errors.